deployment/hasura/metadata/actions.graphql

```diff
@@ -144,6 +151,17 @@ type TypeScriptResponse {
   typescript: String!
 }
 
+type ExpandAllActivitiesResponse {
+  id: Int!
+  expandedActivityInstances: [ExpandedActivityInstance!]!
+}
+
+type ExpandedActivityInstance {
+  id: Int!
+  commands: [Command!]
+  errors: [UserCodeError!]
```

Comment:
@dyst5422 The backend always responds with a non-null list of `errors`, so I was thinking we might as well propagate that through to the DB and this type?

I could see an argument for either though. Having `commands` and `errors` both nullable is "symmetric".

Reply:
Aren't all errors basically TS `UserCodeError`s? If so I don't think it'd be possible to have a successful expansion and a user code error (for a single activity instance expansion).

Reply:
For now that's true, but may not be in the future

Reply:
For example - a user may write a very complicated expansion that handles internal errors and wants to return those errors alongside commands. We could easily add a context function for creating these errors for return alongside the commands.

Reply:
Got it. Maybe you're thinking some day there may be a different class of errors, not just compilation errors.

I'll make the change mentioned in the 1st comment here :)